lib/srv/db/common/errors.go

```diff
@@ -236,3 +236,14 @@ agent's service principal. See: https://goteleport.com/docs/database-access/guid
 		return trace.Wrap(err)
 	}
 }
+
+// IsUnrecognizedAWSEngineNameError checks if the err is non-nil and came from using an engine filter that the
+// AWS region does not recognize.
+func IsUnrecognizedAWSEngineNameError(err error) (res bool) {
+	if err == nil {
+		res = false
+		return
+	}
+	res = strings.Contains(strings.ToLower(err.Error()), "unrecognized engine name")
+	return
+}
```

Comment:
nit: 6 lines vs 4 lines 😄 : 
```suggestion
func IsUnrecognizedAWSEngineNameError(err error) bool {
	if err == nil {
		return false
	}
	return strings.Contains(strings.ToLower(err.Error()), "unrecognized engine name")
}
```

Reply:
oops, this was a leftover of some debugging I was doing with a deferred function to print the result lol. Yes will fix.